docs/workspace/meas/index.md

```diff
@@ -1,2 +1,80 @@
+ a resolution high enough to achieve your desired accuracy of 1mV in a 0-5V range.
+
+
+### Choice for Current measurement:
+
+ - **Hall Effect Sensors:** These devices measure current and voltage through magnetic fields and provide galvanic 
+ isolation. They are suitable for both AC and DC measurements.
+
+**Available option:**
+
+
+- ACS712-20A/ ACS714-20A:
+
+
+[ACS712-20A/ ACS714-20A datasheet]
+
+- TMCS1100:
+
+
+[TMCS1100 datasheet]
+
+The ACS712-20A and the ACS714-20A both serve the same purpose, but the ACS714-20A is better suited for harsher 
+environments. Considering the controlled testing environment and cost factors, I recommend opting for the **ACS712-20A** in this case. 
+
+### Choice of FUSE:
+
+- Thermal Fuse or Switch: This is used to protect the circuit from excessive currents. It should be connected between 
+I_MEAS+ and I_MEAS- and should be able to disconnect the circuit within a specified time if the current exceeds a certain limit.
+
+**Available option:**
+
+  1.	BOURNS CB72ABB: This is a thermal fuse with a functioning temperature of 72°C and a holding temperature of 40°C.
+
+  2.	BOURNS CB85ABB: This thermal fuse has a functioning temperature of 85°C and a holding temperature of 40°C.
+
+  3.	BOURNS SC82AAA: This thermal fuse has a functioning temperature of 82°C and a holding temperature of 40°C.
+
+  4.	BOURNS CB77ABB: This thermal fuse has a functioning temperature of 77°C and a holding temperature of 40°C.
+
+  5.	BOURNS SC85AAB: This thermal fuse has a functioning temperature of 85°C and a holding temperature of 40°C
+
+
+ [Datasheet for fuses]
+
+I would like to go with the below two options because there is ***no need to replace*** them after each time the fuse goes off,
+ unlike thermal fuses or switches.
+
+- **Electronic Fuses (eFuses):** These active devices use a sense resistor and a FET to monitor current flow. When the 
+current exceeds a preset value, the control logic turns the FET off, cutting the flow of current.
+
+Electronic fuses are not suitable for the current in use, as they are typically designed for very small currents. 
+Therefore, circuit breakers would be a better choice
+
+- **Circuit Breakers:** These are resettable switches that open automatically when the current exceeds a certain threshold. 
+They can be manual or automatic and are suitable for repeated use.
+
+**Available option:**
+
+F202 25A-30mA/AC:
+
+[F202 25A-30mA/AC datasheet]
+
+
+
+### Choice for voltage measurement:
+
+
+[TMCS1100 datasheet]: https://www.ti.com/product/TMCS1100#features
+[ACS712-20A/ ACS714-20A datasheet]: https://www.allegromicro.com/en/Products/Sense/Current-Sensor-ICs/Zero-To-Fifty-Amp-Integrated-Conductor-Sensor-ICs/ACS712
+[Datasheet for fuses]: https://www.newark.com/c/circuit-protection/fuses-fuse-accessories/fuses/thermal-fuses?fuse-current=25a
+[F202 25A-30mA/AC datasheet]: https://new.abb.com/products/1SYF202005R1250/in-f202-25a-30ma-ac
```

Comment:
As far as i see that is a [Residual Current Circuit Breaker](https://en.wikipedia.org/wiki/Residual-current_device) (often called `GFCI` in english). It can be used up to $25A$, but it does not break for the total amount of current. It will break if more than $30mA$ of difference between the two conductive paths are detected.
That would not be suitable for our use case.

There is a sort of circuit breakers which have a very good name in german "Leitungs-Schutz-Schalter" which translates to something like "line-safety-switch". Maybe one can call this sort of circuit breaker "line safety circuit breaker". I think this would be the type of circuit breaker that would be best suited for us.
I have selected a suggestion from _ABB_ with a `B25` characteristic:

[ABB line safety circuit breaker - B25](https://new.abb.com/products/de/2CDS251001R0255/s201-b25)